### PR TITLE
Agenda for meetup on Oct 19th

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,29 @@ Join us in [Discord](https://discord.gg/pW35BNSBeV)
...
 # Sydney Meetups (Physical)
 
+## 2023 Oct 19 - 5:30 PM to 8:00 PM AEDT
+
+This meetup will be hosted at MS Reactor, 11 York Street, Lvl 10, RSVP Essential - Spaces limited!
+
+There will be catering in the form of pizza and drinks.
+Thank you to @sigp for sponsoring!
+
+Attendance is limited to XX people.
```
Comment (thomaseizinger) [inline README.md]: Open question: What is our attendance limit?

File: README.md [hunk 2]
```diff
@@ -10,6 +10,29 @@ Join us in [Discord](https://discord.gg/pW35BNSBeV)
...
+
+Please [RSVP on MeetUp](XX) if you come in person!
```
Comment (thomaseizinger) [inline README.md]: Open question: Do we still advertise on meetup?

File: README.md [hunk 3]
```diff
@@ -10,6 +10,29 @@ Join us in [Discord](https://discord.gg/pW35BNSBeV)
...
+No RSVP required if you attend online.
+
+| Time    | Item                   | Description                                          |
+|:--------|:-----------------------|:-----------------------------------------------------|
+| 5:30 pm |                        | Doors open for networking                            |
+| 6:00 pm | Panel discussion       | Proc macros: The good, the bad and the ugly          | 
+| 6:45 pm |                        | BREAK                                                | 
+| 7:00 pm | @voltrevo              | Demo: Valuescript                                    | 
+| 7:30 pm | @christopherhunt       | Demo: Streambed                                      |
```
Comment (thomaseizinger) [inline README.md:31]: Happy with this timing @huntc @voltrevo?

File: README.md [hunk 4]
```diff
@@ -10,6 +10,29 @@ Join us in [Discord](https://discord.gg/pW35BNSBeV)
 
 # Sydney Meetups (Physical)
 
+## 2023 Oct 19 - 5:30 PM to 8:00 PM AEDT
+
+This meetup will be hosted at MS Reactor, 11 York Street, Lvl 10, RSVP Essential - Spaces limited!
+
+There will be catering in the form of pizza and drinks.
+Thank you to @sigp for sponsoring!
```
Comment (thomaseizinger) [inline README.md]:
```suggestion
Thank you to [@sigp](https://github.com/sigp) for sponsoring!
```

**Comment by huntc** [comment]
Sure thing

File: README.md (hunk 3 above)
Comment (voltrevo) [reply README.md:31]: 👍

**Review by pinkforest — APPROVED** [review]
lgtm - it's 80 pax

File: README.md (hunk 1 above)
Comment (thomaseizinger) [reply README.md]:
```suggestion
Attendance is limited to 80 people.
```

File: README.md (hunk 2 above)
Comment (thomaseizinger) [reply README.md]:
```suggestion
Please [RSVP on MeetUp](https://www.meetup.com/rust-sydney/events/296672158/) if you come in person!
```

**Pull request merged**